src/content/app/genome-browser/components/browser-track-config/track-config-views/GeneTrackConfig.tsx

```diff
@@ -0,0 +1,70 @@
+/**
```

Comment:
Do we not want to call them config panels (or settings panels) instead of just configs?

Reply:
created ticket